core/validatorapi/router.go

```diff
@@ -1152,10 +1152,6 @@ func getValidatorIDs(query url.Values) []string {
 
 // getValidatorsByID returns the validators with ids being either pubkeys or validator indexes.
 func getValidatorsByID(ctx context.Context, p eth2client.ValidatorsProvider, stateID string, ids ...string) ([]v1Validator, error) {
```

Comment:
This will fail in `validatorapi` component in `convertValidators` with "pubshare not found"

Reply:
Thanks for the catch 🙏

So we have 2 options I suppose, either only convert the validators that are there in the cluster with the rest of the validators OR just return ALL validators if no indices/pubkeys are specified with no pubkey<>pubshare conversion.

I would prefer the latter one. What do you think?

Reply:
not converting dv public keys will result in the VC not being able to finds it's own validator in the list, resulting in it thinking it is not activated yet. So we have to convert DV public keys to pubshare ALWAYS. 

Suggest somehow detecting that convert should ignore validators if not found in the list. So still erroring by default but allowing only conerting "some" in the case of "querying all validators"